gui/mozregui/pref_editor.py

```diff
@@ -30,13 +30,13 @@ def data(self, index, role=Qt.DisplayRole):
             if index.column() == 0:
                 return name
             else:
-                return repr(value)
+                return QString(value).__str__()
```

Comment:
eh, ok - thanks for pointing this out! Maybe something like ('"%s"' % value) would do it then if the value is of type string ? The reason why I would like to see quotes is because it can be a number, or a string representing a number, e.g "5" vs 5. But that's not that important, so we can leave out the quotes - as you prefer.